src/main/target/STM32F3DISCOVERY/target.h

```diff
@@ -171,6 +171,9 @@
 #define USE_ESCSERIAL
 #define ESCSERIAL_TIMER_TX_PIN  PB8  // (HARDARE=0,PPM)
 
+#define UART2_TX_PIN            PA2
```

Comment:
Don't think this belongs to this PR.

Reply:
Im using it for testing.  Will back it out later.

Reply:
@codecae: Now would be a good time to back it out. ;-)

Reply:
10-4 :)